helpers/msipkg/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/blang/semver/v4"
+)
+
+var (
+	wixTpl = `<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Id="*" Name="gopass" UpgradeCode="{{ .UpgradeCode }}" Language="1033" Codepage="1252" Version="{{ .Version }}" Manufacturer="gopass">
+    <Property Id="PREVIOUSVERSIONSINSTALLED" Secure="yes"/>
+    <Upgrade Id="{{ .UpgradeCode }}">
+      <UpgradeVersion Minimum="0.0.0" Property="PREVIOUSVERSIONSINSTALLED" IncludeMinimum="yes" IncludeMaximum="no"/>
+    </Upgrade>
+    <InstallExecuteSequence>
+      <RemoveExistingProducts Before="InstallInitialize"/>
+    </InstallExecuteSequence>
+    <Package InstallerVersion="200" Compressed="yes" Comments="Windows Installer Package" InstallScope="perUser"/>
+    <Media Id="1" Cabinet="app.cab" EmbedCab="yes"/>
+    <Icon Id="icon.ico" SourceFile="{{ .Icon }}"/>
+    <Property Id="ARPPRODUCTICON" Value="icon.ico"/>
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="LocalAppDataFolder">
+        <Directory Id="INSTALLDIR" Name="gopass">
+          <Component Id="gopass.exe" Guid="*">
+            <File Id="gopass.exe" Source="{{ .Binary }}" Name="gopass.exe"/>
+            <Shortcut Id="StartMenuShortcut" Advertise="no" Icon="icon.ico" Name="gopass" Directory="ProgramMenuFolder" WorkingDirectory="INSTALLDIR" Description=""/>
+            <Shortcut Id="DesktopShortcut" Advertise="no" Icon="icon.ico" Name="gopass" Directory="DesktopFolder" WorkingDirectory="INSTALLDIR" Description=""/>
```

Comment:
I don't see why we would have a start menu or a desktop shortcut for a cmd line app?

Reply:
For me it didn't show up in either place. IIUC thats the effect of `Advertise=no`. But I might be wrong.